test/powershell/engine/Remoting/PSSession.Tests.ps1

```diff
@@ -74,7 +74,10 @@ Describe "SkipCACheck and SkipCNCheck PSSession options are required for New-PSS
         param ($scriptBlock, $expectedErrorCode)
 
         $platformInfo = Get-PlatformInfo
-        if (($platformInfo -eq "alpine") -or ($platformInfo -eq "raspbian")) {
+        if (($platformInfo.Platform -eq "alpine") -or
+            ($platformInfo.Platform -eq "raspbian") -or
+            ($platformInfo.Platform -eq 'centos' -and $platformInfo.Version = '8')
```

Comment:
not skip on Debian here? Just in case you missed the check.

Reply:
Added debian.. good catch.